boxoffice/assets/js/models/util.js

```diff
@@ -28,6 +28,17 @@ export const Util = {
     else {
       return moment(dateTimeString).toString();
     }
+  },
+  getElementId: function(htmlString) {
+    return htmlString.match(/id="(.*?)"/)[1];
+  },
+  getFormTemplate: function(htmlString) {
+    // Add on click event handler for Ractive to submit the form
+    const formTemplate = `${htmlString.slice(0, htmlString.search(/type="submit"/))} on-click="onFormSubmit(event)" {{#formOnSubmit}}disabled{{/}} ${htmlString.slice(htmlString.search(/type="submit"/))}`;
```

Comment:
Can you break this into multiple lines since it's too long? Also you can return the value directly, `formTemplate` isn't really needed.

Reply:
Fixed